cmd/podman/root.go

```diff
@@ -128,6 +128,16 @@ func persistentPreRunE(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	for _, env := range cfg.Engine.Env {
+		splitEnv := strings.SplitN(env, "=", 2)
```

Comment:
Do we want this to take precedence over actual OS environment? I imagine that if I have something set in containers.conf but later do a `ENVVAR=value podman run ...` I want the version passed there to take precedence.

If so, we should only define environment variables that are not already defined.

Reply:
Yes I guess that makes sense.  @QiWang19 Please add a note to the man pages on podman (buildah) about this, also, just do a check to see if environment is set, it set write a debug message, if different, and don't override,